torch/_tensor_docs.py

```diff
@@ -1558,6 +1558,11 @@ def callable(a, b) -> number
 
 Unlike :meth:`~Tensor.expand`, this function copies the tensor's data.
 
+.. warning::
+
+    :func:`torch.repeat` behaves differently from :func:`numpy.repeat`,
+    but is more similar to :func:`numpy.tile`.
```

Comment:
nit: It would be great if we had links to the numpy docs here: https://docs.scipy.org/doc/numpy/reference/generated/numpy.tile.html https://docs.scipy.org/doc/numpy/reference/generated/numpy.repeat.html